src/quo/components/drawers/drawer_buttons/component_spec.cljs

```diff
@@ -6,14 +6,7 @@
     [test-helpers.component :as h]))
 
 (h/describe "drawer-buttons"
-  (h/before-each
-   (fn []
-     (h/use-fake-timers)))
-
-  (h/after-each
-   (fn []
-     (h/clear-all-timers)
-     (h/use-real-timers)))
+  (h/setup-fake-timers)
```

Comment:
FYI: Fake timers are a nice feature. When I tried to enable them in more places, sometimes they could help make the tests more deterministic due to the elimination of some timing issues, but tests ended up failing for other reasons. It's a tool, and as such should be used only when needed.